### PR TITLE
refactor(ecs): rename max/min for ECS variables

### PR DESCRIPTION
**Pull request by iamhopaul123**
----
Rename maximum and minimum to max and min for ECS variables

### Pull Request Checklist

* [x] Testing
  - Unit test added (prefer not to modify an existing test, otherwise, it's probably a breaking change)
  - __CLI change?:__ coordinate update of integration tests with team
  - __cdk-init template change?:__ coordinated update of integration tests with team
* [x] Docs
  - __jsdocs__: All public APIs documented
  - __README__: README and/or documentation topic updated
  - __Design__: For significant features, design document added to `design` folder
* [x] Title and Description
  - __Change type__: title prefixed with **fix**, **feat** and module name in parens, which will appear in changelog
  - __Title__: use lower-case and doesn't end with a period
  - __Breaking?__: last paragraph: "BREAKING CHANGE: <describe what changed + link for details>"
  - __Issues__: Indicate issues fixed via: "**Fixes #xxx**" or "**Closes #xxx**"
* [N/A] Sensitive Modules (requires 2 PR approvers)
  - IAM Policy Document (in @aws-cdk/aws-iam)
  - EC2 Security Groups and ACLs (in @aws-cdk/aws-ec2)
  - Grant APIs (only if not based on official documentation with a reference)

----

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license.

### CODE REVIEW

File: packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-ecs-deploy.expected.json [hunk 1]
```diff
@@ -265,8 +265,8 @@
           "Ref": "EcsCluster97242B84"
         },
         "DeploymentConfiguration": {
-          "MaximumPercent": 200,
-          "MinimumHealthyPercent": 50
+          "maxPercent": 200,
```
Comment (eladb) [inline packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-ecs-deploy.expected.json]: This seems wrong. Was this a bug? I really didn’t expect the CFN to change just the construct APi

Comment (iamhopaul123) [reply packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-ecs-deploy.expected.json]: I didn't mean to make any structure change. Just changed all the variable names containing maximum -> max and minimum -> min.

Comment (iamhopaul123) [reply packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-ecs-deploy.expected.json]: I just want to change the variable names and populate the change to all of their usages. Will that be a problem if I don't make changes to this test files?

Comment (hoegertn) [reply packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-ecs-deploy.expected.json]: You cannot change the names in the `expected` files, as they are defined by the CloudFormation Spec. Why do you want to rename them anyway?

Comment (iamhopaul123) [reply packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-ecs-deploy.expected.json]: Thanks for the feedback. I might need to talk with the team to see if they still want to change those names.

Comment (rix0rrr) [reply packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-ecs-deploy.expected.json]: Here's the thing: you DO want to change the names of the parameters in CDK structs, but you CAN'T change what names they generate to in the CloudFormation template.

Comment (iamhopaul123) [reply packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-ecs-deploy.expected.json]: > Here's the thing: you DO want to change the names of the parameters in CDK structs, but you CAN'T change what names they generate to in the CloudFormation template.

Yeah. I agree with you. I tried to revert the change that I did to the json template, but the unit tests failed.

**Comment by iamhopaul123** [comment]
@pkandasamy91 - please review

**Comment by pkandasamy91** [comment]
Can you rename maxPercent to maxHealthyPercent to maintain consistency btwn min and max.

**Comment by hoegertn** [comment]
I still do not understand why you rename the variables.

1) It is no longer consistent with the CFN property
2) It is less human-friendly 

So what is the reason to rename maximum to max?

**Comment by pkandasamy91** [comment]
> I still do not understand why you rename the variables.
> 
>     1. It is no longer consistent with the CFN property
> 
>     2. It is less human-friendly
> 
> 
> So what is the reason to rename maximum to max?

In order to maintain consistency between all other cdk constructs which are specified using max and min instead of maximum and minimum.

**Pull request merged**